src/plugins/console/server/lib/spec_definitions/js/query/dsl.ts

```diff
@@ -745,6 +745,28 @@ export const query = (specService: SpecDefinitionsService) => {
         // populated by a global rule
       },
     },
+    script_score: {
+      __template: {
+        query: '',
+        script: {},
+      },
+      script: {
```

Comment:
```suggestion
      script: {},
```
`script` should be an empty object because it's also has a global rule for autocompletion [here](https://github.com/elastic/kibana/blob/db8eab9fa3662db535dc170995dd200a5fa5c65b/src/plugins/console/server/lib/spec_definitions/js/globals.ts?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L54)

Reply:
Fixed